physical/raft/fsm.go

```diff
@@ -672,7 +672,7 @@ func (f *FSM) ApplyBatch(logs []*raft.Log) []interface{} {
 							go f.restoreCb(context.Background())
 						}
 					default:
-						return fmt.Errorf("%q is not a supported transaction operation", op.OpType)
+						f.logger.Error("unsupported transaction operation", "op", op.OpType)
```

Comment:
I'm sorry to always be harping on this, but: I feel this could result in an excessive volume of logging.  Won't this mean that nodes not yet upgraded will write a log message for every storage entry that gets modified?  Can we instead log just the first time we see a given unknown optype?

Reply:
Sure

Reply:
Addressed in https://github.com/hashicorp/vault/pull/17732/commits/6246a63cde5d928340ca34dea3aa2b54eb880d0d

Reply:
That only does it for a single batch though, we'd need it to be a field instead of a local for this to make much difference.

Reply:
Ah yes, of course. New version in https://github.com/hashicorp/vault/pull/17732/commits/5a3508e20bcfb44cc8e4a8d20fa3d9fa63693381. I used a `sync.Map` instead of separate map and lock as the documentation mentioned that `sync.Map` is optimized for: `when the entry for a given key is only ever written once but read many times, as in caches that only grow`, which is this case.